src/styles/blocks/_product-card.scss

```
@@ -11,6 +11,11 @@
     width: 160px;
     height: 134px;
     margin: 15px 5px 45px;
+    transition-duration: 0.4s;
```

```suggestion
    transition: transform 0.4s;
```
Always include property that you add transition for.

---

same everywhere